packages/schematics/src/collection/ng-new/files/__directory__/package.json

```diff
@@ -12,6 +12,7 @@
     "affected:apps": "./node_modules/.bin/nx affected:apps",
     "affected:build": "./node_modules/.bin/nx affected:build",
     "affected:e2e": "./node_modules/.bin/nx affected:e2e",
+    "affected:libs": "./node_modules/.bin/nx affected:libs",
```

Comment:
Could you also take care of `ng-add` please? The same line needs to be added here: https://github.com/nrwl/nx/blob/master/packages/schematics/src/collection/ng-add/index.ts#L62